app/app/github.py

```diff
@@ -45,12 +45,15 @@ def search(q):
     return response.json()
 
 
-def get_issue_comments(owner, repo):
+def get_issue_comments(owner, repo, issue=None):
```

Comment:
couldn't find any existing usages of this function

method signature remains consistent but allows for retrieving comments of a single issue

Reply:
👍 Some of the functionality has changed in the GH integration PR.